packages/calcite-components/src/utils/sortableComponent.ts

```diff
@@ -1,10 +1,14 @@
 import Sortable from "sortablejs";
 const sortableComponentSet = new Set<SortableComponent>();
 
-export interface DragDetail {
-  toEl: HTMLElement;
-  fromEl: HTMLElement;
-  dragEl: HTMLElement;
+export interface DragDetail<
```

Comment:
@jcfranco is this what you were thinking? Will this type the elements correctly?

Reply:
It should if you provide the types in the emitter type.

```ts
@Event({ cancelable: false }) calciteListOrderChange: EventEmitter<
  DragDetail<
    HTMLCalciteListElement, 
    HTMLCalciteListElement, 
    HTMLCalciteListItemElement
  >
>;
```